x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/fix_logs_step/fix_logs_step.test.tsx

```diff
@@ -370,4 +373,29 @@ describe('Overview - Fix deprecation logs step', () => {
       });
     });
   });
+
+  describe('Privileges check', () => {
+    test(`It doesn't show analyze and resolve logs if it doesn't have the right privileges`, async () => {
+      await act(async () => {
+        testBed = await setupOverviewPage({
+          privileges: {
+            hasAllPrivileges: false,
+            missingPrivileges: {
+              index: [DEPRECATION_LOGS_INDEX],
+            },
+          },
+        });
+      });
+
+      const { exists, component } = testBed;
+
+      component.update();
+
+      // No index privileges warning callout should be shown
+      expect(exists('noIndexPermissionsCallout')).toBe(true);
```

Comment:
Do we have any tests that assert this isn't shown when the user has the required privileges?

Reply:
Nice call, I've just added one that checks for that also